database/migrations/2022_11_07_171403_add_config_descriptions.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+return new class() extends Migration {
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		DB::table('configs')->where('key', 'version')->update(['description' => 'Current version of Lychee']);
+		DB::table('configs')->where('key', 'check_for_updates')->update(['description' => 'Automatically check for new updates']);
+		DB::table('configs')->where('key', 'sorting_photos_col')->update(['description' => 'Default column used for sorting photos']);
+		DB::table('configs')->where('key', 'sorting_photos_order')->update(['description' => 'Default order used for sorting photos']);
+		DB::table('configs')->where('key', 'sorting_albums_col')->update(['description' => 'Default column used for sorting albums']);
+		DB::table('configs')->where('key', 'sorting_albums_order')->update(['description' => 'Default order used for sorting albums']);
+		DB::table('configs')->where('key', 'imagick')->update(['description' => 'Enable/Disable imagick processing']);
+		DB::table('configs')->where('key', 'dropbox_key')->update(['description' => 'Dropbox API key']);
+		DB::table('configs')->where('key', 'skip_duplicates')->update(['description' => 'Skip duplicate if found on import']);
+		DB::table('configs')->where('key', 'lang')->update(['description' => 'Language used by Lychee']);
+		DB::table('configs')->where('key', 'layout')->update(['description' => 'Layout for pictures']);
+		DB::table('configs')->where('key', 'default_license')->update(['description' => 'Default License used for albums']);
+		DB::table('configs')->where('key', 'small_max_width')->update(['description' => 'Maximum width for small thumbs (justified album view)']);
+		DB::table('configs')->where('key', 'small_max_height')->update(['description' => 'Maximum height for small thumbs (justified album view)']);
```

Comment:
```suggestion
		DB::table('configs')->where('key', 'small_max_width')->update(['description' => 'Maximum width for small thumbs ((un)justified album view)']);
		DB::table('configs')->where('key', 'small_max_height')->update(['description' => 'Maximum height for small thumbs ((un)justified album view)']);
```
It's both justified and unjustified. I don't know if I like my own suggestion, but something needs to be done.